README.md

```diff
@@ -4,14 +4,14 @@
 [![Travis CI](https://img.shields.io/travis/temando/remark-mermaid.svg)](https://travis-ci.org/temando/remark-mermaid)
 [![MIT License](https://img.shields.io/github/license/temando/remark-mermaid.svg)](https://en.wikipedia.org/wiki/MIT_License)
 
-Replaces [`mermaid`](https://mermaidjs.github.io/) graphs with rendered SVGs.
+Replaces graphs defined in [`mermaid`](https://mermaidjs.github.io/) with rendered SVGs.
```

Comment:
Maybe it'd be more accurate to say:

Replaces fenced code blocks in [`mermaid`](https://mermaidjs.github.io/) format with:

- Links to rendered SVG files of the graph (default mode).
- Mermaid-formatted code wrapped in `div` tags for rendering by `mermaidjs` (simple mode).

Reply:
Good!